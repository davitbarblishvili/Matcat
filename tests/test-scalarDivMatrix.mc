// Author: Davit
func main() int {
    matrix a; 
    matrix b; 

    a = [[1,2,3],[4,5,6],[2,10,2]];
    b = a / 2;

    printm(b);

    return 0;

}