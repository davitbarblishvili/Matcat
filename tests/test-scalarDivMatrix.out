(
[0.50 1.00 1.50 ]
[2.00 2.50 3.00 ]
[1.00 5.00 1.00 ]
)